src/variables.js

```diff
@@ -152,10 +151,9 @@ module.exports = {
 				if (JSON.stringify(oldValue) !== JSON.stringify(newValue)) {
 					this.store[key] = newValue
 					if (key === 'clips') {
+						this.initVariables()
```

Comment:
Had to initialize variables as well to represent the new amount of clips passed from playoutbee